lib/features/give/widgets/numeric_keyboard.dart

```diff
@@ -135,70 +142,42 @@ class _NumericKeyboardState extends State<NumericKeyboard> {
     );
   }
 
-  Expanded _buildCommaButton() => Expanded(
-        child: InkWell(
-          borderRadius: BorderRadius.circular(45),
-          onTap: widget.leftButtonFn,
-          child: Container(
-            margin: const EdgeInsets.all(4),
-            padding: const EdgeInsets.all(12),
-            alignment: Alignment.center,
-            decoration: BoxDecoration(
-              color: Colors.white,
-              borderRadius: BorderRadius.circular(5),
-            ),
-            child: Text(
-              ',',
-              style: TextStyle(
-                fontSize: 18,
-                fontWeight: FontWeight.bold,
-                color: widget.textColor,
-              ),
-            ),
-          ),
-        ),
-      );
-
-  Expanded _buildDeleteButton() => Expanded(
-        child: InkWell(
-          borderRadius: BorderRadius.circular(45),
-          onTap: widget.rightButtonFn,
-          child: Container(
-            margin: const EdgeInsets.all(4),
-            padding: const EdgeInsets.all(12),
-            alignment: Alignment.center,
-            decoration: BoxDecoration(
-              color: Colors.white,
-              borderRadius: BorderRadius.circular(5),
-            ),
-            child: widget.rightIcon,
+  Widget _calcButton(
+    String value, {
+    void Function()? onTap,
+    Widget? child,
+  }) {
+    final onButtonTap = onTap ?? () => widget.onKeyboardTap(value);
+    final valueWidget = child ??
+        Text(
+          value,
+          textAlign: TextAlign.center,
+          style: TextStyle(
+            fontSize: 18,
+            fontWeight: FontWeight.bold,
+            color: widget.textColor,
           ),
-        ),
-      );
-
-  Widget _calcButton(String value) {
+        );
+    final size = MediaQuery.sizeOf(context);
+    var padding = 14.0;
+    if (size.height < 700) {
```

Comment:
what is 700?

Reply:
In this context is the screen height. If the screen height is less than 700 then we add that padding. I thought it was self-explanatory as I used the Media query of size